spark-doris-connector/src/main/java/org/apache/doris/spark/load/DorisStreamLoad.java

```diff
@@ -112,6 +113,7 @@ public DorisStreamLoad(SparkSettings settings) {
         this.db = dbTable[0];
         this.tbl = dbTable[1];
         this.fenodes = settings.getProperty(ConfigurationOptions.DORIS_FENODES);
+        this.beNodes = settings.getProperty(ConfigurationOptions.DORIS_BENODES);
```

Comment:
Can this be changed to lowercase? Unify with fenodes

Reply:
ok